app/src/main/res/layout/fragment_edit_contest_category.xml

```diff
@@ -0,0 +1,36 @@
+<?xml version="1.0" encoding="utf-8"?>
+<LinearLayout xmlns:android="http://schemas.android.com/apk/res/android"
+              android:layout_width="match_parent"
+              android:layout_height="match_parent"
+              android:layout_margin="@dimen/activity_vertical_margin"
+              android:orientation="vertical"
+    >
+
+    <android.support.design.widget.TextInputLayout
```

Comment:
Why do you use TextInputLayout and TextInputEditText?
(Curious because I created a custom view for EditText (with cross button to clear all text) and might incorporate it in your layouts.)